server/routes/allocation.js

```diff
@@ -0,0 +1,85 @@
+const express = require('express');
```

Comment:
Having files called both `allocation` and `phase-allocation` is confusing, as are the endpoints of the same name. I'd propose coming up with a better distinction between the two - perhaps `phase-allocation` is the actual allocation stuff (as you are doing here), and the global stuff becomes `global-phases` or something? I'm open to ideas on how to rename these.

Reply:
This is exactly what I wanted feedback on - it was getting confusing for me too. I originally started naming everything 'PhaseAllocations' to match the db table/FK relationship (without using SitePhaseAllocation because that seemed too long). But eventually, I simplified everything to just 'allocation' as there appeared to be some overlap with global phases (Which has a few different names in the codebase -ie  phase/phases, sitePhases, globalPhases)

I can rename a few things - I kind of think allocations and phases can stand alone without any extra context in their names. 
ie - createAllocation, updateAllocation, getPhase, createPhase, updatePhase. Thoughts?

Reply:
I think `phases` vs. `allocations` would be the way to go. I wouldn't think global phase, because there's a chance it won't be fully "global" later on.

Reply:
If this is the agreed path forward, I can rename all the allocation stuff in this PR, and create a separate PR/ticket to rename phases.

Reply:
That works for me!